### ISSUE TITLE
The spotlight is lost if user scrolls the page

### ISSUE THREAD

**Issue by alvaromontoro** [alvaromontoro/spotlightjs#3]
SpotlighJS was designed for an unchangeable/unscrollable viewport. If the plugin is used on a regular web page and the user scrolls, the spotlight is lost. It would be nice if the spotlight never disappears from the current viewport.

**Comment by dulcetti** [comment]
Hi @alvaromontoro. Can I take this too? What do you think if on user start the Spotlight, the scrolling turn off, and if he exit of Spotlight, the scrolling come back?

**Comment by alvaromontoro** [comment]
@dulcetti Of course. I was thinking about doing something like adding `overflow:hidden` to the html/body and readjusting the scroll so the user didn't notice a jump when the animation started or ended... but it may be overcomplicating things. Any suggestion is welcome.
